File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.java
class Solution {
    public int largestAltitude(int[] gain) {
        int leftSum = 0;
        int highestAlt = 0;
        
        for(int i=0; i<gain.length; i++) {
            leftSum += gain[i];
            highestAlt = Math.max(highestAlt, leftSum);
        }
        
        return highestAlt;
    }
}

File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.cs
public class Solution {
    public int LargestAltitude(int[] gain) {
        int leftSum = 0;
        int higheshAlt = 0;
        for(int i=0; i<gain.Length; i++) {
            leftSum += gain[i];
            higheshAlt = Math.Max(higheshAlt, leftSum);
        }
        return higheshAlt;
    }
}